lib/games_statistics.rb

```diff
@@ -0,0 +1,112 @@
+require "CSV"
+require_relative "./games"
+require_relative "./teams"
+require_relative "./game_teams"
+
+class GamesStatistics
+  attr_reader :games, :game_teams, :teams
+  
+  def self.from_csv(locations)
+    GamesStatistics.new(locations)
+  end
+
+  def initialize(locations)
```

Comment:
Yeah, this would dry up this code for the csv's but still give us access to the objects.